Formula/faas-cli.rb

```diff
@@ -0,0 +1,26 @@
+class FaasCli < Formula
+  desc "CLI for templating and/or deploying FaaS functions"
+  homepage "http://docs.get-faas.com/"
+  url "https://github.com/alexellis/faas-cli/archive/0.3.tar.gz"
+  sha256 "d4db7ddb0ea1dadf469c3da9893b004874d19570d09457e8cf86e5aef0df00f5"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["XC_OS"] = "darwin"
+    ENV["XC_ARCH"] = MacOS.prefer_64_bit? ? "amd64" : "386"
+    ENV["GOPATH"] = buildpath
+    faas_cli_path = buildpath/"src/github.com/alexellis/faas-cli/"
+    faas_cli_path.install Dir["*"]
+
+    cd "src/github.com/alexellis/faas-cli/" do
+      system "go", "build"
+      bin.install faas_cli_path/"faas-cli"
+    end
+  end
+
+  test do
+    system bin/"faas-cli", "-action", "build"
+    assert_match "Give a valid -image name for your Docker image.", shell_output("#{bin}/faas-cli -action build")
```

Comment:
if I use `buildpath` if fails as that variable isn't initialised within the test block
```
undefined method `/' for nil:NilClass
```